src/Student10/Ex4/Tweet.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+
+ 
+export const Tweet = (props) => {
+        
+  return (
+    <div className='Tweet' style={{ backgroundColor: props.color }}>
+        {props.message}
+        <button onClick={() => props.onHandleClick(props.message * 2)}> Like</button> 
```

Comment:
There's no reason to pass `props.message * 2` into this callback, and it doesn't make sense.  Why would we multiply the message times two?

Reply:
Removed it